website/docs/auth/service_principal_client_secret.html.markdown

```diff
@@ -111,41 +117,98 @@ $ az vm list-sizes --location westus
 $ az account list-locations
 ```
 
+Finally, since we're logged into the Azure CLI as a Service Principal we recommend logging out of the Azure CLI (but you can instead log in using your user account):
+
+```bash
+$ az logout
+```
+
+Information on how to configure the Provider block using the newly created Service Principal credentials can be found below.
+
+---
+
 ### Creating a Service Principal in the Azure Portal
 
-There are two tasks needed to create a Service Principal via [the Azure Portal](https://portal.azure.com):
+There are three tasks needed to create a Service Principal via [the Azure Portal](https://portal.azure.com):
```

Comment:
```suggestion
There are three tasks that need to be completed in order to create a Service Principal via [the Azure Portal](https://portal.azure.com):
```

Reply:
```suggestion
There are three tasks required to create a Service Principal via [the Azure Portal](https://portal.azure.com):
```
?

Reply:
closing in favour of the suggestion below